.gitlab-ci.yml

```diff
@@ -1,29 +1,12 @@
 ---
 include:
   - project: nci-gdc/gitlab-templates
-    ref: 0.6.2
+    ref: master
```

Comment:
Doesn't this need to be a specific version number?

Reply:
After @kulgan's update, it should be kinda stable now.